package.json

```diff
@@ -251,22 +248,27 @@
     "rollup-plugin-esbuild": "^6.1.0",
     "shx": "^0.3.4",
     "typescript": "^5.3.2",
+    "use-sync-external-store": "^1.2.0",
     "vitest": "^0.34.6"
   },
   "peerDependencies": {
-    "immer": ">=9.0",
     "@types/react": ">=16.8",
-    "react": ">=16.8"
+    "immer": ">=9.0",
```

Comment:
Yarn prefers to have these dependencies listed in alphabetical order which is why these changed.